test_util/convertECL.cpp

```diff
@@ -293,6 +293,8 @@ int main(int argc, char **argv)
 
     std::string rootN = filename.substr(0,p);
     std::string extension = filename.substr(p,l-p);
+    std::transform(extension.begin(), extension.end(), extension.begin(),
+                 [](unsigned char ckey){ return ::toupper(ckey);});  
```

Comment:
Now I wonder: How does this compile without `std` in front of the colons? (And why did you not write std::toupper in the first place.)

Reply:
> How does this compile without `std` in front of the colons

When including the `<cX>` C library compatibility headers, an implementation is **permitted**, but not **required**, to export the C standard library functions in the global namespace in addition to the `std::` namespace.  Similarly, when including the `<X.h>` headers, an implementation is **permitted**, but not **required**, to export the C standard library functions in the `std::` namespace in addition to the global namespace.  User code should not rely on this, however, and should generally refer to `std::c_function()` when including `<cX>` and use the undecorated name `c_function()` when including `<X.h>`.

Reply:
I was not familiar with that. I will add the std:: tomorrow.